waffle_hub/dataset/dataset.py

```diff
@@ -48,10 +48,12 @@ class Dataset:
     EXPORT_DIR = Path("exports")
     SET_DIR = Path("sets")
     DRAW_DIR = Path("draws")
+    BACKGROUND_RAW_DIR = RAW_IMAGE_DIR / Path("background")
```

Comment:
It can cause confliction with some raw images whose path starts with `background`.

We can resolve this issue by make `raw_image_dir / labeled`, `raw_image_dir / unlabeled`, `raw_image_dir / background`. But, this solution is hard to be compatible with older version (migration issue).

For the future migration, I suggest to make `BACKGROUND_RAW_DIR` to be just `Path("background")`.

Reply:
Currently, the base path when reading images from other methods is 'raw_image_dir'.
So if 'background' is not placed in 'raw_image_dir', file_name may get twisted in json.

As an alternative, I will add a check code for image duplication.